index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    
+    <header class="nav-bar">
+           <a class="logo" href="#">My logo</a>
+           <a class="menu-button" href="#"><img src="/images/Menu.png" alt=""></a>
```

Comment:
- [ ] [REQUIRED] Please ensure you stick to the design as much as possible. The hamburger button is missing on the right. So please ensure you add it and it should be in a `nav` tag, not an `a` tag